packages/translate/src/index.js

```diff
@@ -164,12 +164,14 @@ Translate.prototype.detect = function(input, callback) {
 /**
  * Get an array of all supported languages.
  *
- * @resource [Discover Supported Languages]{@link https://cloud.google.com/translate/v2/using_rest#supported-languages}
+ * @resource [Discovering Supported Languages]{@link https://cloud.google.com/translate/v2/discovering-supported-languages-with-rest}
  *
+ * @param {string=} target - Include the languages names in the response
```

Comment:
Oops, I misunderstood what the `target` option was for. If the purpose is to get more information from the API, I feel bad not giving it directly to the user as you originally were.

So, this is what I would expect:

```js
translate.getLanguages('en', function(err, languages) {
  languages = [
    {
      code: 'af', // changed key from "language" to "code"
      name: 'Afrikaans'
    },
    // ...
  ]
})
```

I think `code` is easier for this case:

```js
translate.getLanguages('en', function(err, languages) {
  languages.forEach(function(language) {
    var code = language.language; // `language.language` :(
  })
})
```

Sorry for the frustration I've surely caused :\